### PR TITLE
Added printer-friendly styling to README

### LINKED ISSUE #24: Add printer-friendly style
Add a custom style that is more friendly to printers.

### PR DESCRIPTION
**Pull request by martijnvankekem**
See issue #24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,25 @@ A modern, clean, and very simple responsive HTML invoice template, because somet
...
 
+## Printer-friendly styling
+By default, the layout of the invoice when printed is equal to the regular layout. Optionally, the box-shadow and border line can be removed when printing, to give a more invoice-like feeling.
+<p align="center">
+  <img alt="Without printer friendly styling" src="https://user-images.githubusercontent.com/22718003/147985007-fb298dd8-b88c-4407-bdea-26fe2c9f595c.png" width="45%">
+&nbsp; &nbsp; &nbsp; &nbsp;
+  <img alt="With printer-friendly styling" src="https://user-images.githubusercontent.com/22718003/147985049-ebf70654-ace3-4e49-b7e1-9fb4d3583163.png" width="45%">
+</p>
+
+Example:
```
Comment (WesCossick) [inline README.md]: I'm a bit picky when it comes to verbiage. I'd reword this and remove the screenshot to reduce clutter:

```suggestion
By default, the appearance of the invoice when printed is the same as when viewed on a screen. If the invoice is the primary or only element on the page, you may want to consider removing the box-shadow and border, and letting the invoice extend the full width of the page. That can be accomplished with this CSS:
```

File: README.md [hunk 2]
```diff
@@ -19,6 +19,25 @@ A modern, clean, and very simple responsive HTML invoice template, because somet
...
+```
```
Comment (WesCossick) [inline README.md]: This code block should specify the language, to get the correct color-coding:

```suggestion
```css
```

**Review by WesCossick — CHANGES_REQUESTED** [review]
Thanks for submitting the PR! I've suggested a few changes.

**Comment by martijnvankekem** [comment]
I understand. I've accepted and committed the proposed changes.

**Review by WesCossick — APPROVED** [review]
LGTM!
